content/general/concepts/data-types/data-types.md

```diff
@@ -21,9 +21,9 @@ In computer programming, primitive data types are the building blocks for any va
 <!-- prettier-ignore -->
 | Data Type | Description |
 | --- | --- |
-| Number | Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. |
-| String | Strings are collections of characters text. Depending on the language, single quotes (`''`) or double quotes (`""`) can be used. For example: `"Codecademy is awesome! 🖥️⌨️"` |
-| Boolean | Booleans hold a `true` or `false` value. |
+| [Number](https://www.codecademy.com/resources/docs/general/number) | Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. |
+| [String](https://www.codecademy.com/resources/docs/general/string) | Strings are sequences of characters. Depending on the language, single quotes (`''`) or double quotes (`""`) can be used. For example: `"Codecademy is awesome! 🖥️⌨️"` |
```

Comment:
Think we should add (<code>``</code>) to this?

Reply:
You mean mention backticks? Sure! Maybe something like this:

```
Strings are sequences of characters surrounded by single quotes (''), double quotes (""), and/or backticks (``), depending on the language. For example: `"Codecademy is awesome! 🖥️⌨️"`
```

Reply:
Exactly

Reply:
@Dusch4593 can I use HTML in the `.md` file. That's the only way (that I know of) to format the backticks.

Reply:
What about putting a forward slash in front of each one? 

\` like this? \`

Reply:
Didn't know I can do that. I was going to use the `<code>` tag.

Reply:
Yeah I would recommend using the backslash for two reasons:

* Keep it all Markdown.
* Your most recent change, I worry, might end up not translating correctly if this file is ultimately minified in some way (which will get rid of your whitespace and leave us with 4 backticks in a row.).